javascript/extractor/src/com/semmle/js/extractor/FileExtractor.java

```diff
@@ -104,7 +104,7 @@ private static int skipBOM(byte[] bytes, int length) {
 
   /** Information about supported file types. */
   public static enum FileType {
-    HTML(".htm", ".html", ".xhtm", ".xhtml", ".vue", ".hbs", ".ejs", ".njk") {
+    HTML(".htm", ".html", ".xhtm", ".xhtml", ".vue", ".hbs", ".ejs", ".njk", ".erb") {
```

Comment:
Perhaps restrict to `.html.erb`. While many `erb` files are html-like templates, they are also used for many other text formats.

```suggestion
    HTML(".htm", ".html", ".xhtm", ".xhtml", ".vue", ".hbs", ".ejs", ".njk", ".html.erb") {
```